### ISSUE TITLE
Veronte / Veronte / Ubxdev : LEA-6T utilizar la señal de timepulse.

### ISSUE THREAD

**Issue by gir2** [gir2/prueba_issues#16]
**Reported by admin on 5 Mar 2013 15:46 UTC**
None

**Comment by gir2** [comment]
**Modified by dbs on 28 Aug 2013 12:13 UTC**

**Comment by gir2** [comment]
**Modified by dbs on 28 Aug 2013 12:17 UTC**

**Comment by gir2** [comment]
**Comment by dbs on 28 Aug 2013 12:22 UTC**
sincronizacion t-GPS tow, UBX mensajes de tiempo, etc.

**Comment by gir2** [comment]
**Comment by dbs on 28 Aug 2013 17:37 UTC**
Clock session initialization should use GPS instead of Pctime.

**Comment by gir2** [comment]
**Comment by dbs on 2 Sep 2013 07:58 UTC**
Ver Timing_AppNote_(GPS.G6-X-11007) de u-blox.
Dice que no hay que utilizarlo a menos de 1Hz

**Comment by gir2** [comment]
**Comment by jbm <jbm@embention.com> on 25 Sep 2013 15:13 UTC**
In [3dcfe4ffd641cad0eb7577270c8ed6a3eb0dc30f/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="3dcfe4ffd641cad0eb7577270c8ed6a3eb0dc30f"
Refs #13, closes #15.
 - Added extra configuration to ECAP to allow it to work as GPS time pulse.

Refs #236:
 - Time precission internally working at cpu clock precission (150MHz clock).
 - Clock and Chrono adapted.

Refs #227:
 - Rawdat now uses 64 bits Real for both carrier and pseudoranges (removed d2fp, and subfp) instead of Uint64.
 - Serialization of RAW from GND changed (values out of str range found on previous version).

Changed JFX project properties.
```
